pkg/scm/git/clone_test.go

```diff
@@ -16,9 +16,10 @@ func TestCloneWithContext(t *testing.T) {
 	c := &Clone{gh, fs}
 
 	fakeConfig := &api.Config{
-		Source:     "https://foo/bar.git",
-		ContextDir: "subdir",
-		Ref:        "ref1",
+		Source:           "https://foo/bar.git",
+		ContextDir:       "subdir",
+		Ref:              "ref1",
+		DisableRecursive: true,
```

Comment:
@liggitt Please read what I write on the pull request :)

I'll quote again...

```
@bparees Test is failing because the last command is now the submodule update instead of the checkout. I can fix this a few different ways. I think I'll add another test for the submodule update.
```

As with recursive the last command (the only command actually checked in the test) is `submodule update --init` etc, the test fails. I disable recursive so that the last command is the checkout, which is what the test is checking for. It's checking the same thing. Just not the submodule checkout, which wasn't tested before anyway.